M2/Macaulay2/packages/Macaulay2Doc/doc.m2

```diff
@@ -274,7 +274,8 @@ document {
 	  prepended to the path, based on the value of the ", TO "applicationDirectory", "
 	  for your system."
 	  },
-     PRE concatenate between_"\n" apply(value Core#"private dictionary"#"userpath",s -> (5,s)),
+     PRE replace(regexQuote homeDirectory, "/home/m2user/",
```

Comment:
Why not use `~` instead here?

Reply:
It seemed more realistic since running the actual code prints the full path:

```m2
i12 : concatenate between_"\n" apply(value Core#"private dictionary"#"userpath",s -> (5,s))

o12 =      /home/profzoom/.Macaulay2/code/
           /home/profzoom/.Macaulay2/local/share/Macaulay2/
```